### PR TITLE
Remove `IntKey` and `TimestampKey`

### PR DESCRIPTION
**Pull request by ueco-jb**
closes https://github.com/CosmWasm/cw-plus/issues/570

### CODE REVIEW

File: packages/storage-plus/src/de.rs [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-#![allow(deprecated)]
-
 use std::array::TryFromSliceError;
 use std::convert::TryInto;
 
 use cosmwasm_std::{Addr, StdError, StdResult};
 
-use crate::int_key::CwIntKey;
-use crate::keys::{IntKey, TimestampKey};
+use crate::int_key::IntKey;
```
Comment (maurolacy) [inline packages/storage-plus/src/de.rs:6]: I'm a little worried about this renaming introducing confusion. The module name is different, though. Perhaps that's enough to disambiguate / clarify this change.

**Review by maurolacy — APPROVED** [review]
Looks good.

Only doubt I have is with renaming `CwIntKey` in this PR / release round.

We'll leave `IntKeyOld` for a while as well. So, perhaps doing the renaming above when removing `IntKeyOld` is a good idea.

Approving anyway, as I think this is a viable option.

File: packages/storage-plus/src/de.rs (hunk 1 above)
Comment (ueco-jb) [reply packages/storage-plus/src/de.rs]: Had similar doubts.
All right, I reverted that commit - now it could be cherry-picked some time in the future.

Comment (maurolacy) [reply packages/storage-plus/src/de.rs]: Now tracked in #775.

**Pull request merged**